gstd/src/common/handlers.rs

```diff
@@ -26,55 +26,25 @@
 //! For `debug` mode it provides more extensive logging.
 
 #[cfg(target_arch = "wasm32")]
-use core::{alloc::Layout, arch::wasm32, panic::PanicInfo};
+use core::panic::PanicInfo;
 
-#[cfg(not(feature = "debug"))]
-#[cfg(not(debug_assertions))]
-#[cfg(target_arch = "wasm32")]
-#[alloc_error_handler]
-pub fn oom(_: Layout) -> ! {
-    wasm32::unreachable()
-}
-
-#[cfg(any(feature = "debug", debug_assertions))]
-#[cfg(target_arch = "wasm32")]
-#[alloc_error_handler]
-pub fn oom(_: Layout) -> ! {
-    crate::debug!("Runtime memory exhausted. Aborting");
-    wasm32::unreachable()
-}
-
-#[cfg(not(feature = "debug"))]
-#[cfg(not(debug_assertions))]
-#[cfg(target_arch = "wasm32")]
-#[panic_handler]
-pub fn panic(_: &PanicInfo) -> ! {
-    wasm32::unreachable();
-}
-
-#[cfg(any(feature = "debug", debug_assertions))]
```

Comment:
Sorry cannot agree with that though it provides better diagnostics.
It will lead to our programs begin to swell in size.

Reply:
relates to #1857